templates/edit.php

```diff
@@ -88,17 +88,24 @@
       </select>
     </td>
   </tr>
+  <?php if(isset($user_data['wprdesknumber'][0]) && $user_data['wprdesknumber'][0] != 'Off-Site') {?>
```

Comment:
The data gets sync'd into LDAP and while null & 'Off-Site' do functionally mean the same thing with regard to output in phonebook we don't have control ingress to LDAP to set wprdesknumber == null if wprdesknumber == 'Off-Site'. There's also a good chance that this attribute will be used via external LDAP reporting where the differentiation between null and 'Off-Site' could be important.